protostuff-parser/src/main/antlr3/io/protostuff/parser/ProtoParser.g

```diff
@@ -113,6 +116,12 @@ annotation_keyval [Proto proto, Annotation annotation]
         )
     ;
 
+doc_entry [Proto proto]
+    :   DOC {
+            proto.addDoc($DOC.text.substring(3).trim());
```

Comment:
You did not understand my point. I already have comments like this in my code:

```
/// Message description
/// Usage:
/// <code>
/// class X extends Foo<Bar> {
///     public static final int BAZ = 1;
///     int z() {
///         // ...
///     }
/// }
/// </code>
```

This comment should be transformed to java comment:

```
/**
 * Message description
 * Usage:
 * <code>
 * class X extends Foo<Bar> {
 *     public static final int BAZ = 1;
 *     int z() {
 *         // ...
 *     }
 * }
 * </code>
 */
```

Your solution is completely ok, the only thing I need is to remowe that `.trim()` call.